pyne/mcnp.py

```diff
@@ -1523,7 +1539,6 @@ def mat_from_inp_line(filename, mat_line, densities='None'):
 
     return finished_mat
 
-
 class Wwinp(Mesh):
```

Comment:
```suggestion

class Wwinp(Mesh):
```
I believe PEP8 requires 2 blank lines bewteen classes